Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -33,16 +33,16 @@ namespace Google.Apis.Auth.OAuth2
     /// See <see cref="GetApplicationDefaultAsync(CancellationToken)"/> for the credential retrieval logic.
     /// </para>
     /// </summary>
-    public class GoogleCredential : ICredential, ITokenAccessWithHeaders, IOidcTokenProvider
+    public class GoogleCredential : IOidcTokenProvider, IAcceptsQuotaProject
```

Comment:
I'm assuming at least one of those implements ICredential and ITokenAccessWithHeaders

Reply:
Yes, but as chatted offline, I'll remove IAcceptsQuotaProject and this won't apply. This change will be reverted (and similar ones).